administration/src/components/generation/CardCreationModel.ts

```diff
@@ -6,15 +6,15 @@ export interface CardCreationModel {
     id: number,
     forename: string,
     surname: string,
-    expirationDate: Date,
+    expirationDate: Date | null,
```

Comment:
I think this should not be nullable after our last discussion.

A new flag should be introduced: `infiniteValid`